documentation/home.md

```diff
@@ -0,0 +1,30 @@
+# MuleSoft AI Chain Connector Overview
+
+The MuleSoft AI Chain Connector helps developers design, build and manage AI agents within Anypoint Platform. It provides the tools and support needed to integrate Large Language Models (LLMs), vector stores, and other advanced AI services into MuleSoft applications.
```

Comment:
The MuleSoft AI Chain Connector enables developers to orchestrate agentic experiences from within Anypoint Platform. It provides the tools and support needed to integrate Large Language Models (LLMs), vector stores, and other advanced AI services into MuleSoft applications.